libs/error-stack/macros/Cargo.toml

```diff
@@ -1,25 +1,21 @@
 [package]
-name = "error-stack-macros"
-version = "0.0.0-reserved"
-authors = { workspace = true }
+name = "error-stack-derive"
```

Comment:
Could you elaborate why you rename a crate owned by us? The crate is [published on `crates.io`](https://crates.io/crates/error-stack-macros) already. You did not even mention this in the PR description!

Reply:
That's an oversight, I'll revert